src/client/app/shared/safe-image.component.ts

```diff
@@ -14,9 +15,10 @@ export class SafeImageComponent {
 
 	@Input() showDefault: boolean = true;
 
-	private defaultPicSrc: string;
+	defaultPicSrc: string;
```

Comment:
Since you're fixing things in here, I noticed that the useDefaultUrl function is private. I think that's gonna cause an issue with the AOT build because the template references it.

Also, the configService is not referenced by the template, so it can be private.